indico/modules/events/registration/templates/management/regform.html

```diff
@@ -1,16 +1,18 @@
 {% extends 'events/registration/management/_regform_base.html' %}
 
 {% block content %}
-    {% if not regform.has_started %}
-        {% include 'events/registration/management/regform_actions/not_started.html' %}
-    {% elif regform.is_open %}
-        {% include 'events/registration/management/regform_actions/open.html' %}
-    {% elif regform.has_ended %}
-        {% include 'events/registration/management/regform_actions/ended.html' %}
-    {% endif %}
+    {% block regform_state %}
+        {% if not regform.has_started %}
+            {% include 'events/registration/management/regform_actions/not_started.html' %}
+        {% elif regform.is_open %}
+            {% include 'events/registration/management/regform_actions/open.html' %}
+        {% elif regform.has_ended %}
+            {% include 'events/registration/management/regform_actions/ended.html' %}
+        {% endif %}
+    {% endblock %}
 
-    <div class="action-box">
-        {% block regform_setup %}
+    {% block regform_setup %}
```

Comment:
To give a bit more context, I would consider naming this block `regform_setup_box`:

```suggestion
    {% block regform_setup_box %}
```